templates/_nav.html

```diff
@@ -1,42 +1,47 @@
-<div class="navbar navbar-default">
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
   <a class="navbar-brand">
-    <img src="/static/movr-symbol.png" height="30">
+    <img src="/static/img/movr-symbol.png" height="40" width="40">
   </a>
   {% if available %}
-  <div class="navbar">
-    <ul class="nav navbar-nav">
-      <li>
-        <a href="/">Home</a>
+
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse" id="navbarNavDropdown">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="/">Home</a>
```

Comment:
Note that https://github.com/cockroachlabs/movr-flask/pull/13 removes the Home page